src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -59,6 +66,27 @@ if(coreHasFeature(REFERENCE_ICON_USING_STRING)) {
     expect(vm.$el.tagName).toBe('svg')
     expect(vm.$el.classList.contains('fa-coffee')).toBeTruthy()
   })
+
+  test('using a FASS icon with string format', () => {
+    const vm = mountFromProps({ icon: 'fass fa-arrow-left'})
+  
+    expect(vm.$el.tagName).toBe('svg')
+    expect(vm.$el.classList.contains('fa-arrow-left')).toBeTruthy()
+  })
+
+  test('using a sharp solid icon long prefix with array format', () => {
```

Comment:
```suggestion
  test('using a sharp solid icon long prefix with string format', () => {
```

I believe this would be string format?

Reply:
Yep. Good catch.